doc/docs/master/enterprise/auth/authorization/index.md

```diff
@@ -42,54 +42,42 @@ Pachyderm has a number of predefined roles granting permissions to its Resources
 
-    !!! Info
-        When Pachyderm Auth is not enabled,
-        all users are clusterAdmins.
+- **clusterAdmin**: A clusterAdmin can perform any action on the cluster.
+Activating auth (`pachctl auth activate`) creates a Root User with irrevocable `clusterAdmin` rights. This Role must be set at the cluster level only.
 
-    The `clusterAdmin` role is broken up in a set of finer grained roles, so users can delegate specific tasks without giving full admin privileges to a cluster.
-    Those roles are listed below. The union of them all is equivalent to the ClusterAdmin role.
-
-    !!! Note
-        None of those sub-roles have shared privileges.
+- **secretAdmin**: A secretAdmin has the ability to create, update, delete Kubernetes secrets on a cluster.
 
 - **idpAdmin**: An ipdAdmin can configure identity providers. They can perform operations such as `pachctl idp create-connector` or `pachctl idp update-connector`.
 
-- **identityAdmin**: An identityAdmin has the ability to configure the identity service.  In particular, they can set an issuer: `pachctl idp set-config --issuer http://<some IP>:658` ?? //TODO What are the pachctl command available to the identityAdmin?
+- **identityAdmin**: An identityAdmin has the ability to configure the identity service. They can perform operations like `pachctl idp set-config` and `pachctl idp get-config`.  
 
-- **debugger**: A debugger role has the ability to produce debug dumps.
+- **debugger**: A debugger has the ability to produce debug dumps using `pachctl debug`.
 
-- **licenseAdmin**: This role grant the ability to register new clusters with the license server, as well as manage and update the enterprise license. For example, this role can perform a `pachctl enterprise register`, `pachctl license activate` or `pachctl license delete-cluster`. //TODO get validation
+- **licenseAdmin**: This role grant the ability to register new clusters with the license server, as well as manage and update the enterprise license. For example, this role can perform a `pachctl enterprise register`, `pachctl license activate` or `pachctl license delete-cluster`. 
 
-- **oidcAppAdmin**: An oidcAppAdmin can configure oidc apps between the Identity service and a cluster. In other terms, they can perform operations such as `pachctl idp create-client`. This command is rarely necessary to most users. Usually, a `pachctl deploy` will invoke a `pachctl idp create-client` under the hood and register Pachyderm's apps like the IDE, for example. However, there might be cases where this registration of new oidc clients needs do be made explicitely. This could happen in the case were someone decides to reinstall Pachyderm manually, component by component, without the automated `pachctl deploy` for example???????????. //TODO Uhm.... So not clear. Get a clearer picture of when this create-client command would be useful, when it is implicitly called, and what other commands this role has access to. 
```

Comment:
> Usually, a `pachctl deploy` will invoke a `pachctl idp create-client` under the hood and register Pachyderm's apps like the IDE, for example

You're right that usually users won't need to explicitly configure the OIDC apps with `pachctl idp create-client`, but they do still need this permission because the permissions happen at the RPC level. So this role is needed to deploy dash or activate auth in pachd.